app/com/arpnetworking/pillar/PillarInitializer.java

```diff
@@ -67,32 +66,41 @@ public PillarInitializer(
     }
 
     private void runMigrations() {
-        for (Map.Entry<String, Configuration.DatastoreConfig> entry : _configuration.getDatastores().entrySet()) {
+        for (final Map.Entry<String, Configuration.DatastoreConfig> entry : _configuration.getDatastores().entrySet()) {
             final String dbName = entry.getKey();
             final Configuration.DatastoreConfig config = entry.getValue();
 
             final String directory = config.getDirectory();
-            if (!Strings.isNullOrEmpty(directory)) {
-                final URL resource = _environment.resource(directory);
-                if (resource != null) {
-                    LOGGER.info("Running migrations for Cassandra database " + dbName);
-                    try {
-                        final scala.collection.immutable.Map<String, Object> replication = JavaConverters.mapAsScalaMapConverter(
-                                config.getReplication()).asScala().toMap(Predef.conforms());
-                        final File file = new File(resource.toURI());
-                        final Registry registry = Registry.fromDirectory(file);
-                        final Migrator migrator = new CassandraMigrator(registry);
-                        final Session session = _injector.getInstance(Key.get(Session.class, Names.named(dbName)));
-                        session.init();
-                        migrator.initialize(session, config.getKeyspace(), new ReplicationOptions(replication));
-                        session.execute("USE " + config.getKeyspace());
-                        migrator.migrate(session, Option.<Date>none().asScala());
-                    } catch (final URISyntaxException e) {
-                        LOGGER.error("Unable to run migrations at " + resource, e);
-                    }
-                } else {
-                    LOGGER.info("Found no migrations in directory " + directory + " for Cassandra database " + dbName);
+            final URL resource = _environment.resource(directory);
```

Comment:
Uh... ok.... I didn't think we were annotating local variables, just variables in interfaces.